pyscriptjs/src/exceptions.ts

```diff
@@ -2,15 +2,10 @@ const CLOSEBUTTON = `<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16'
 
 type MessageType = 'text' | 'html';
 
-/*
-These error codes are used to identify the type of error that occurred.
-The convention is:
-* PY0 - errors that occur when fetching
-* PY1 - errors that occur in config
-* Py2 - errors that occur in plugins
-* PY9 - Deprecation errors
-*/
-
+/**
+ * These error codes are used to identify the type of error that occurred.
+ * TODO: describe in which case these are useful (or for who) as these are never used or shown.
+ */
```

Comment:
You are probably not done yet. I'm just thinking out loud while I glance over the PR. Yes, we should describe the error code categories, who are these for here but also add to the docs (arguably more important actually :) )

Reply:
We do have these in the docs under exceptions but I think we may not have all of them.

When I added this docstring the idea was to keep the reference in the code itself so devs can have a look and know which ones to raise.

With that said, I don't think we should remove them from the code :)

Reply:
Makes sense. That's what I first thought when I saw this change but I'm not too strongly opinionated (only that it should most definitely be in the docs). In general, 2 the options have pros and cons:

* having the codes here and in the docs means we need to make sure we are always updating in 2 places (which, in my experience, is really prone to error)
* having the codes only on the docs and having a link to the docs in the code (the info is less obvious to see from the source code but if a user wants, they can open docs)

Between the 2 options, I'm +0 keeping it on the docs only and having a link in the code (because, in most cases, comments can easily get outdated)

Reply:
I've updated the code to point at https://docs.pyscript.net/latest/reference/exceptions.html?highlight=errors which indeed seems like a very good idea